### PR TITLE
Week-3-PR-2: Add graph using a CSV file and the Google Charts Library

### PR DESCRIPTION
**Pull request by gracieorozco**
I added a graph using a CSV file with the Google Charts Library.

### CODE REVIEW

File: portfolio/src/main/webapp/projects.html [hunk 1]
```diff
@@ -121,7 +118,12 @@ <h4>Computer Science Fundamentals Website</h4>
                 </p>
             </li>
         </ul>
-      </p>
+    </p>
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/projects.html]: I didn't find a matching opening tag for </p>. Shall we remove this?

File: portfolio/src/main/webapp/projects.html [hunk 2]
```diff
@@ -121,7 +118,12 @@ <h4>Computer Science Fundamentals Website</h4>
                 </p>
             </li>
         </ul>
-      </p>
+    </p>
+    <h2>Google Charts Library</h2>
+    <p>Graphs made from the Google Charts Library</p>
+    <div id="chart-container"></div>
+    <div id="chart-csv-container"></div>
+    <p>Resource: http://www.ecdms.energy.ca.gov/elecbycounty.aspx</a></p>
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/projects.html]: I don't see a matching opening tag for <a>. Did you want to make the URL a hyperlink?

File: portfolio/src/main/webapp/WEB-INF/ElectricityByCounty.csv [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+SACRAMENTO,4713.027288
+SAN BENITO,124.535133
+SAN BERNARDINO,5443.731723
+SAN DIEGO,6942.482091
+SAN FRANCISCO,1462.547442
+SAN JOAQUIN,1816.39498
+SAN LUIS OBISPO,644.755402
+SAN MATEO,1510.175756
+SANTA BARBARA,777.567635
+SANTA CLARA,3860.162566
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/WEB-INF/ElectricityByCounty.csv:44]: I live here. This was really good idea for a dataset for creating a chart. Good job!

File: portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/graph-data")
```
Comment (KidEinstein) [inline portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java:32]: Let's add a javadoc above this class to briefly specify the use of this class.

**Review by KidEinstein — APPROVED** [review]
Thanks, Gracie! This PR looks good to me. Left a few minor comments and approving the PR. Please address the comments before merging. 

Also, make sure to merge this into master *after* you have merged week-3-pr-1 into master.

File: portfolio/src/main/webapp/projects.html (hunk 1 above)
Comment (gracieorozco) [reply portfolio/src/main/webapp/projects.html]: Removed extra tag.

File: portfolio/src/main/webapp/projects.html (hunk 2 above)
Comment (gracieorozco) [reply portfolio/src/main/webapp/projects.html]: Added extra opening tag.

File: portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java (hunk 4 above)
Comment (gracieorozco) [reply portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java:32]: Added a javadoc.

**Review by KidEinstein — APPROVED** [review]
Thanks for updating!

**Pull request merged**